src/app.tsx

```diff
@@ -1,4 +1,4 @@
-import { MainLayout } from 'layout/main';
+import { AppRouter } from 'components/router/router';
```

Comment:
```suggestion
import { AppRouter } from 'components/router';
```
если есть директория с компонентом, у нее должен быть публичный api (реэкспорт через index)

Reply:
забыла, да, сделала index